atomic_reactor/plugins/pre_download_remote_source.py

```diff
@@ -58,6 +58,22 @@ def get_remote_source_config(self, session, url, insecure=False):
         response.raise_for_status()
         return response.json()
 
+    def generate_cachito_env_file(self):
+        """
+        Generate cachito.env file with exported environment variables received from
+        cachito request.
+        """
+
+        if self.buildargs:
+            self.log.info('Create %s file with environment variables '
+                          'received from cachito request', CACHITO_ENV_FILENAME)
+
+            abs_path = os.path.join(REMOTE_SOURCE_DIR, CACHITO_ENV_FILENAME)
+            with open(abs_path, 'w') as f:
+                f.write('#!/bin/bash\n')
+                for env_var, value in self.buildargs:
+                    f.write('export {}={}\n'.format(env_var, value))
```

Comment:
It might be a good idea to quote the value, maybe with [shlex.quote()](https://docs.python.org/3/library/shlex.html#shlex.quote)?

Reply:
Good idea, added.